src/storage/file_io.c

```diff
@@ -10393,6 +10398,63 @@ fileio_restore_volume (THREAD_ENTRY * thread_p, FILEIO_BACKUP_SESSION * session_
 
 	      fileio_dismount (thread_p, prev_vdes);
 	    }
+
+	  // *INDENT-OFF*
+	  auto find_unlinked_vol = [&volid] (const tuple <int, string, string> &unlinked_vol)
+	  {
+	    int unlinked_volid = get<0> (unlinked_vol);
+
+	    if (unlinked_volid == volid)
+	      {
+		return true;
+	      }
+	    else
+	      {
+		return false;
+	      }
+	  };
+	  // *INDENT-ON*
+
+	  if (incremental_includes_volume_header == true)
+	    {
+	      if (volid != LOG_DBFIRST_VOLID && is_prev_vol_header_restored == false)
+		{
+		  _er_log_debug (ARG_FILE_LINE, "RESTOREDB: [FOUND UNLINK] [lv%d] volid=%d, vol=%s, prev_vol=%s\n",
+				 session_p->dbfile.level, volid, to_vol_label_p, prev_vol_label_p);
+
+		  auto it_found = find_if (unlinked_volinfo.cbegin (), unlinked_volinfo.cend (), find_unlinked_vol);
```

Comment:
We usually use standard libraries with `std::` prefix to distinguish it instead of using `using namespace std;` above.
```suggestion
		  auto it_found = std::find_if (unlinked_volinfo.cbegin (), unlinked_volinfo.cend (), find_unlinked_vol);
```